x-pack/plugins/security_solution/public/flyout/entity_details/components/user_details_content.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { useExpandableFlyoutContext } from '@kbn/expandable-flyout';
+import { EuiFlyoutBody } from '@elastic/eui';
+import type { RiskScoreState } from '../../../explore/containers/risk_score';
+import type {
+  ManagedUserData,
+  ObservedUserData,
+} from '../../../timelines/components/side_panel/new_user_detail/types';
+import type { RiskScoreEntity } from '../../../../common/risk_engine';
+import { FlyoutLoading } from '../../shared/components/flyout_loading';
+import { UserDetailsBody } from './user_details_body';
+import { FlyoutNavigation } from '../../shared/components/flyout_navigation';
+import { RiskInputsPanelKey } from '../../risk_inputs';
+
+export interface UserDetailsContentProps extends Record<string, unknown> {
+  userName: string;
+  observedUser: ObservedUserData;
+  managedUser: ManagedUserData;
+  riskScoreState: RiskScoreState<RiskScoreEntity.user>;
+  contextID: string;
+  scopeId: string;
+  isDraggable: boolean;
+}
+
+/**
+ * This is a visual component. It doesn't access any external API.
+ * It designed for unit testing the UI and previewing changes on storybook.
+ */
+export const UserDetailsContent = ({
+  observedUser,
+  managedUser,
+  contextID,
+  scopeId,
+  userName,
+  isDraggable,
+  riskScoreState,
+}: UserDetailsContentProps) => {
+  const { data: userRisk } = riskScoreState;
+  const userRiskData = userRisk && userRisk.length > 0 ? userRisk[0] : undefined;
+
+  const { openLeftPanel } = useExpandableFlyoutContext();
+  const openPanel = useCallback(() => {
+    openLeftPanel({
+      id: RiskInputsPanelKey,
+      params: {
+        riskInputs: userRiskData?.user.risk.inputs,
+      },
+    });
+  }, [openLeftPanel, userRiskData?.user.risk.inputs]);
+
+  if (riskScoreState.loading || observedUser.isLoading || managedUser.isLoading) {
+    return <FlyoutLoading />;
+  }
+
+  return (
+    <>
+      <FlyoutNavigation flyoutIsExpandable={!!userRiskData?.user.risk} expandDetails={openPanel} />
+      <EuiFlyoutBody>
```

Comment:
you can use the new [FlyoutBody](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/shared/components/flyout_body.tsx) component here instead of `EuiFlyoutBody`. It already contains a `EuiPanel` for you

Reply:
That would complicate the implementation of the modal. It has horizontal bars that take 100% of the width. `FlyouBody` has margins by default and it will demand a creative solution involving negative margins.

<img width="522" alt="283049318-2a1cc223-6fc3-426f-a282-f00ce2b9a042" src="https://github.com/elastic/kibana/assets/1490444/3394fa6a-03db-40fa-b881-acbe8f504343">